src/Microsoft.PowerShell.ConsoleHost/host/msh/ProgressNode.cs

```diff
@@ -336,6 +356,88 @@ private static void RenderFullDescription(string description, string indent, int
                     maxWidth));
         }
 
+        internal static bool IsMinimalProgressRenderingEnabled()
+        {
+            return ExperimentalFeature.IsEnabled("PSAnsiProgress") && PSStyle.Instance.Progress.View == ProgressView.Minimal;
```

Comment:
Why don't we use constants for feature names? "PSAnsiProgress" looks like a puzzle.

Reply:
Good question, we haven't been doing this.

Reply:
I'll update it for this feature in this PR, but won't update the other ones.  We can follow this pattern going forward.